index.js

```diff
@@ -259,14 +268,16 @@ module.exports = function(THREE) {
                 // restrict phi to be betwee EPS and PI-EPS
                 phi = Math.max( EPS, Math.min( Math.PI - EPS, phi ) );
 
+                scale += scaleDelta * (( this.enableDamping === true ) ? this.dampingFactor : 1);
```

Comment:
I suggested renaming scale to scaleDelta and operating with that. Now with these 2 properties I do not trust that the animation really tries to reach the desiredZoomScale and there are no side effects when user starts using mouse wheel when the animation is still in progress. Would like to see that in action and explore how it works.